### PR TITLE
Upgrade to Python 3.8 to catch up with latest project-template changes.

### PR DESCRIPTION
**Pull request by martinpovolny**
## Related Issues and Dependencies

Reflects
https://github.com/aicoe-aiops/project-template/pull/30
https://github.com/aicoe-aiops/ocp-ci-analysis/pull/140

## This introduces a breaking change

- [ ] Yes
- [x] No

It specifies python 3.8, we have 3.6 in the Pipfile, need to fix this.

## This Pull Request implements

Update to the latest changes in the project-template

## Description

As explained in the related PR as "some minor updates to the CI configuration" probably the main change is the new Python version.

### CODE REVIEW

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # Setup and configuring aicoe-ci with configuration file `.aicoe-ci.yaml`
+# b
```
Comment (tumido) [inline .aicoe-ci.yaml:2]: Is this intentional or a blip?

**Review by tumido — COMMENTED** [review]
1 comment, other than that

/lgtm

**Review by tumido — COMMENTED** [review]
/lgtm

seems like a conflict somewhere

**Comment by martinpovolny** [comment]
Yes, the same was done here: https://github.com/aicoe-aiops/ocp-ci-analysis/pull/140

Except for updating the Pipfiles to 3.8.

**Comment by martinpovolny** [comment]
I did not notice that @goern created the PR while also pinging in the channel about updating projects. My bad.

But we should update the Pipfiles to 3.8 anyway. Shouldn't we?

**Comment by tumido** [comment]
@martinpovolny I don't think it's necessary for you to upgrade to Python 3.8. On the other hand if there's anything actually holding you back I see no reason why shouldn't you upgrade. :) So I'd say it's up to you.

**Comment by martinpovolny** [comment]
@tumido : are you saying that it's ok to have python 3.8 in thoth's config and use 3.6 in the Pipfile?

**Comment by tumido** [comment]
@martinpovolny I think you should remain consistent. Precommit for python 3.6 behaves differently to precommit 3.8 for example - because the language is different a tiny bit. So if you require your app to use Python 3.6 in Pipfile, you should stick with 3.6 in thoth's config. But if there's nothing blocking you to upgrade your Pipfile to 3.8, I'd say you fully upgrade to 3.8.

**Review by harshad16 — CHANGES_REQUESTED** [review]
Good to see, project migrating to python 3.8
This is to be updated https://github.com/aicoe-aiops/ocp-ci-analysis/blob/42412bbe6dc98592edecd0c2568bb75eaa127b6e/.aicoe-ci.yaml#L9 with quay.io/thoth-station/s2i-custom-py38-notebook:latest

**Comment by martinpovolny** [comment]
@harshad16 : thanks for the explanation. I hoped to get to some meaningful error myself but when clicking on the "build-check" link above I get only "Error loading PipelineRun" and "PipelineRun not found"

And this is shortly after I requested the `/rebuild` so the data should not have been removed yet.

**Comment by harshad16** [comment]
I hoped to get to some meaningful error myself but when clicking on the "build-check" link above I get only "Error loading PipelineRun" and "PipelineRun not found"

This happens sometimes, for me readload works 
we would work on upgrading the dashboard as well
actually we had migrated the build to operate-first last week, but due to the upgrade of operate-first , we switched the app to psi
once the operate-first is available , we would have more flexibility to uprgade and provide the services in more efficient manner.

**Pull request closed without merging**